#160747738 Improve the responsiveness of the UI

AwaMelvine opened the pull request:
#### What does this PR do?
Improves the responsiveness of the UI template pages.

#### Description of Task to be completed?
- Split the CSS code into multiple files
- Changed colors of buttons to blend with the overall theme
- Make tables horizontally scrollable on smaller screens
- Make the footer to cover the whole screen on devices with smaller screens

#### How should this be manually tested?
Run the UI on the browser and open the developer tools.
Click on the screens icon to view on smaller screens. Continuously
shrink the width of the small interface that appears while observing 
who the interface responds accordingly.

#### What are the relevant pivotal tracker stories?
[#160747738] Split CSS code into different files
[#160748067] Homepage image/intro should fill screen height
[#160737399] Change the button colors to match the overall theme of the site
[#160747595] Fix footer alignment and tables display issues on mobile
[#160775411] Make items on menu page to be 3 per row and centralized
[#160747818] Improve tables styling and buttons responsiveness
[#160749526] Use smaller font sizes for email addresses listed on the tables

AwaMelvine committed:
The single styling file in use now is too long.
split it into multiple files and link it in the
appropriate files

AwaMelvine committed:
The colors for the button should blend well with
those of the overal theme of the website.

AwaMelvine committed:
- The tables should be horizontally scrollable on mobile
- The footer should be slightly centralized on larger as
well as smaller screens

AwaMelvine committed:
- Reduce the font size of the emails listed on the tables
- Use suggestive colors on the text representing the status
of an order

AwaMelvine committed:
Merge pull request #18 from AwaMelvine/ch-ui-for-larger-screens-160788631

160788631 Ch ui for larger screens

robocopkaka commented:
Really like the improvements you've made to the UI. Could you try to center `Manage Orders` in-between the first two divs in the below image
<img width="960" alt="screen shot 2018-09-26 at 1 35 38 pm" src="https://user-images.githubusercontent.com/11176000/46080239-4feab280-c191-11e8-8599-f83b1ec8fa19.png">

AwaMelvine committed:
Use a grid system of displaying the menu instead of
a vertical menu bar

AwaMelvine committed:
[#160748152] Emphasize dashboard menu items

Add hover effect with appropriate colors. Also do a few
final touches on the footer

[Fixes #160748152]

AwaMelvine committed:
[#160775411] Provide some padding around food items

Each food item should have a padding of at least 5px
on the homepage

[Fixes #160775411]

AwaMelvine committed:
[#160788631] Write media queries for larger screens

The footer should stick to the bottom on larger screens.
The text on the page and on buttons should be readable.

[Fixes#160788631]

AwaMelvine committed:
[#160747738] Fix dashboard menu grid layout

Ensure that when wrapping, the bottom item should
be centralized

[Fixes #160747738]

AwaMelvine committed:
Merge branch 'gh-pages' of https://github.com/AwaMelvine/fast-food-fast into gh-pages
